agent/firefoxextensions.ps1

```diff
@@ -86,14 +86,10 @@ if ((test-path 'C:\Program Files (x86)\Mozilla Firefox\firefox.exe') -or (test-p
                 } 
                       
             } 
-            else 
-  {
-    $xml += "<BROWSEREXTENSIONS />"
-  }
         } 
     } 
- 
-  
+} else {
+	$xml += "<BROWSEREXTENSIONS />"
 }
 #[Console]::OutputEncoding = [System.Text.Encoding]::UTF8 
```

Comment:
@Lea9250 ,

By default this line should be uncommented :-)

Regards,
Charlene